src/reducers/FilterUtils.js

```diff
@@ -0,0 +1,60 @@
+export const handleSortByPrice = (items, { sortBy }) => {
+    switch (sortBy) {
+      case "ascending":
+        return items.sort(
+          (firstItem, secondItem) => firstItem.price - secondItem.price
+        );
+      case "descending":
+        return items.sort(
+          (firstItem, secondItem) => secondItem.price - firstItem.price
+        );
+      default:
+        return items;
+    }
+  };
+
+export const handleFilterByRating = ( items , {rating}) =>  {
+    const filteredItems = items.filter(product => product.rating >= rating);
+    //console.log(filteredItems);
```

Comment:
Do remove the console.logs before the final merge.

Reply:
Yes thats what I planning to do. I need logs for debugging right now.